### ISSUE TITLE
Is it possible to read counters?

### ISSUE THREAD

**Issue by cmcqueen** [mullvad/nftnl-rs#44]
See [Counters — Listing named counters from nft command line](https://wiki.nftables.org/wiki-nftables/index.php/Counters#Listing_named_counters_from_nft_command_line) in the nftables wiki.

That has an example of reading a single counter:

    % nft list counter inet named_counter_demo cnt_http

I basically want to do the equivalent of that in Rust code. Is that possible with this crate?

**Comment by MarkusPettersson98** [comment]
No, and we currently have no plans on adding such functionality.
